backend/src/appointment/database/repo/availability.py

```diff
@@ -0,0 +1,77 @@
+"""Module: repo.availability
+
+Repository providing CRUD functions for availability database models.
+"""
+
+from sqlalchemy.orm import Session
+from .. import models, schemas, repo
+
+
+""" AVAILABILITY repository functions
+"""
+
+
+def get(db: Session, availability_id: int) -> models.Availability | None:
+    """retrieve availability by id"""
+    if availability_id:
```

Comment:
I'm sure this is just a copy and paste, but since we're typing as an int this will only get called if you pass in 0.

Reply:
Although I guess this makes sense 🤔

Reply:
This is how we do it in all repo get functions. I thought this is exactly to handle the 0 case, since ids always start at 1... Or maybe I don't understand the problem here 😂

Reply:
just thinking out loud, all good here!